tasks/grafana_config.yml

```diff
@@ -31,6 +42,26 @@
     notify:
       - Restart grafana server
 
+
+  # - name: Copy grafana plugins from local
```

Comment:
This task can be refactor to use search like in `grafana_search_config.yml` so that if in playbook's `files` directory have plugins directory, this task will run.